scripts/mac_intel_install.sh

```diff
@@ -64,9 +64,9 @@ echo "${prefix}Reloading zsh"
 source ~/.zshrc
 
 
-echo "${prefix}Attemping to change pyenv version to 3.10.2"
-pyenv install 3.10.2
-pyenv global 3.10.2
+echo "${prefix}Attemping to change pyenv version to 3.11.0"
+pyenv install 3.11.0
```

Comment:
sorry to keep picking at this - can we do 3.11.1 here too

Reply:
okay updated